tests/integration/workflows/test_workflows_tasks_actions.py

```diff
@@ -428,12 +373,47 @@ def test_refextract_from_pdf(
     mocked_is_pdf_link,
     mocked_package_download,
     mocked_arxiv_download,
-    insert_cited_record,
     workflow_app,
     mocked_external_services
 ):
-    """Test refextract from PDF by going through the entire workflow."""
+    """Test refextract from PDF and reference matching for default Configuration
+     by going through the entire workflow."""
+
+    cited_record_json = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        '_collections': ['Literature'],
+        'abstracts': [
+            {
+                'source': 'arXiv',
+                'value': 'The effects on the non-relativistic dynamics of a system compound by two electrons interacting by a Coulomb potential and with an external harmonic\r\noscillator potential, confined to move in a two dimensional Euclidean space, are investigated. In particular, it is shown that it is possible to determine\r\nexactly and in a closed form a finite portion of the energy spectrum and the associated eigeinfunctions for the Schrodinger equation describing the relative motion of the electrons, by putting it into the form of a biconfluent Heun equation. In the same framework, another set of solutions of this type can\r\nbe straightforwardly obtained for the case when the two electrons are submitted also to an external constant magnetic field.'
+            }
+        ],
+        'arxiv_eprints': [
+            {
+                'categories': ['quant-ph', 'cond-mat.mes-hall', 'cond-mat.str-el', 'math-ph', 'math.MP'],
+                'value': '1308.0815'
+            }
+        ],
+        'authors': [
+            {'full_name': 'Caruso, F.'},
+            {'full_name': 'Martins, J.'},
+            {'full_name': 'Oguri, V.'},
+        ],
+        'control_number': 1000,
+        'document_type': ['article'],
+        'dois': [
+            {'value': '10.1016/j.aop.2014.04.023'}
+        ],
+        'titles': [
+            {
+                'source': 'arXiv',
+                'title': 'Solving a two-electron quantum dot model in terms of polynomial solutions of a Biconfluent Heun equation'
+            }
+        ],
+    }
 
+    TestRecordMetadata.create_from_kwargs(
```

Comment:
Here I would `create_from_file`, so that that big record above doesn't interrupt the flow of the test.

Reply:
Yes, my recommendation would be to either have short records with strictly the necessary information in the body of the test, or long records loaded as fixtures (for example, for regressions that require complicated sets of data to be triggered).